plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php

```diff
@@ -0,0 +1,773 @@
+						],
+					],
+				],
+			]
+		);
+		$pricing_advanced_block = $product_pricing_section->add_block(
+			[
+				'id'         => 'product-pricing-advanced',
+				'blockName'  => 'woocommerce/product-collapsible',
+				'attributes' => [
+					'toggleText'       => __( 'Advanced', 'woocommerce' ),
+					'initialCollapsed' => true,
+					'persistRender'    => true,
+				],
+			]
+		);
+		$pricing_advanced_block->add_block(
+			[
+				'id'         => 'product-tax-class',
+				'blockName'  => 'woocommerce/product-radio-field',
+				'attributes' => [
+					'title'       => __( 'Tax class', 'woocommerce' ),
+					'description' => sprintf(
+					/* translators: %1$s: Learn more link opening tag. %2$s: Learn more link closing tag.*/
+						__( 'Apply a tax rate if this product qualifies for tax reduction or exemption. %1$sLearn more%2$s.', 'woocommerce' ),
+						'<a href="https://woocommerce.com/document/setting-up-taxes-in-woocommerce/#shipping-tax-class" target="_blank" rel="noreferrer">',
+						'</a>'
+					),
+					'property'    => 'tax_class',
+					'options'     => [
+						[
+							'label' => __( 'Standard', 'woocommerce' ),
+							'value' => '',
+						],
+						[
+							'label' => __( 'Reduced rate', 'woocommerce' ),
+							'value' => 'reduced-rate',
+						],
+						[
+							'label' => __( 'Zero rate', 'woocommerce' ),
+							'value' => 'zero-rate',
+						],
+					],
+				],
+			]
+		);
+	}
+
+	/**
+	 * Adds the inventory group blocks to the template.
+	 */
+	private function add_inventory_group_blocks() {
+		$inventory_group = $this->add_group(
+			[
+				'id'         => 'inventory',
+				'order'      => 30,
+				'attributes' => [
+					'id'    => 'inventory',
+					'title' => __( 'Inventory', 'woocommerce' ),
+				],
+			]
+		);
+		$inventory_group->add_block(
+			[
+				'id'         => 'product_variation_notice_inventory_tab',
+				'blockName'  => 'woocommerce/product-has-variations-notice',
+				'attributes' => [
+					'id'         => 'wc-product-notice-has-options',
+					'content'    => __( 'This product has options, such as size or color. You can now manage each variation\'s price and other details individually.', 'woocommerce' ),
+					'buttonText' => __( 'Go to Variations', 'woocommerce' ),
+					'type'       => 'info',
+				],
+			]
+		);
+		// Product Pricing Section.
+		$product_inventory_section       = $inventory_group->add_section(
+			[
+				'id'         => 'product-inventory-section',
+				'attributes' => [
+					'title'       => __( 'Inventory', 'woocommerce' ),
+					'description' => sprintf(
+					/* translators: %1$s: Inventory settings link opening tag. %2$s: Inventory settings link closing tag.*/
+						__( 'Set up and manage inventory for this product, including status and available quantity. %1$sManage store inventory settings%2$s', 'woocommerce' ),
+						'<a href="' . admin_url( 'admin.php?page=wc-settings&tab=products&section=inventory' ) . '" target="_blank" rel="noreferrer">',
+						'</a>'
+					),
```

Comment:
Any reason why `sprintf` was used instead of `createInterpolateElement`?

Reply:
Same reason as above: https://github.com/woocommerce/woocommerce/pull/39814#discussion_r1302177468